desc/vmec.py

```diff
@@ -355,12 +362,21 @@ def save(cls, eq, path, surfs=128, verbose=1):
 
         iotaf = file.createVariable("iotaf", np.float64, ("radius",))
         iotaf.long_name = "rotational transform on full mesh"
-        iotaf[:] = eq.iota(r_full)
+        if eq.iota is not None:
+            iotaf[:] = eq.iota(r_full)
+        else:
+            # value closest to axis will be nan
```

Comment:
@ddudt does this nan at axis in `iota` affect anything? should we add a change to overwrite in #262? VMEC does this (ensures 0 gradient at axis by doing something with the half-grid iota) 
![image](https://user-images.githubusercontent.com/37969854/191356729-bc3b8d48-4b2e-482d-a5cc-74b5ebbcbe6d.png)
according to [educational_vmec](https://github.com/jonathanschilling/educational_VMEC/blob/ff7b1ab6924729034c06846b668c7d23c470c1b1/src/add_fluxes.f90#L67)

Reply:
This is only an issue with computing with a linear grid. I have not noticed nan's at axis for concentric or quadrature with this [test](https://github.com/PlasmaControl/DESC/blob/c9a9d225535f1e10ba52c62238b9a6b96f0372a3/tests/test_constrain_current.py#L69).

Reply:
thing is neither of those grids have a point at `rho=0` right? so I think it would be a nan here since we need to give a value at `rho=0`.

Reply:
It's not a problem as far as I'm aware. I think the best solution would be to computing the equivalent power series for the iota profile and evaluating that at the axis (using a quad grid). But this can be saved for a future PR (such as the vmec one).